metricbeat/module/kafka/consumergroup/consumergroup.go

```diff
@@ -111,6 +112,10 @@ func (m *MetricSet) Fetch(r mb.ReporterV2) {
 	}
 
 	emitEvent := func(event common.MapStr) {
+		// Helpful IDs for dashboards
```

Comment:
I would rather describe it as helpful for queries. 

For the future: I wonder if this is work that should be done in the processors to have it config based instead of in the code.